app/views/schools/alert_reports/index.html.erb

```diff
@@ -64,12 +64,11 @@
   <div class="row padded-row">
```

Comment:
Reword the message when an alert hasn't run as it looks like it's giving the reason why the alert hasn't run rather than which alert it is...